### PR TITLE
Minor JS tweaks found with ESLint.

### PR DESCRIPTION
**Pull request by XhmikosR**
BTW, I still believe the current JSHint settings are weird, but I guess the final call is for @alanshaw to make :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,7 +28,7 @@ var app = express()
 
 app.engine("html", consolidate.handlebars)
 app.set("view engine", "html")
-app.set("views", __dirname + "/dist")
```
Comment (ilanbiala) [inline index.js]: why does it need to be `path.join()`ed?

**Comment by ilanbiala** [comment]
What about semicolons?

File: index.js (hunk 1 above)
Comment (XhmikosR) [inline index.js]: Look my previous comment.

**Comment by XhmikosR** [comment]
Not my choice; that's what is already being used.

**Comment by ilanbiala** [comment]
@alanshaw can semicolons be added in?

**Comment by XhmikosR** [comment]
@alanshaw: bump

**Pull request closed without merging**